source/_components/sensor.hp_ilo.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+title: "HP ILO"
+description: "How to integrate HP ILO (Integrated Lights-Out) sensors within Home Assistant."
+date: 2016-08-15 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hewlett_packard_enterprise.png
+ha_category: Sensor
+ha_release: pre 0.27
```

Comment:
You can change this to 0.27. This branch will not be made public until after the release.